src/MRA.Jobs.Client/Pages/Admin/UserRoles.razor

```diff
@@ -0,0 +1,27 @@
+﻿@page "/UserRoles/{Username}"
```

Comment:
Can you set authorize attribute here?

Reply:
authorization check pass in the mra.jobs.api becouse i sending request before loading page